web/cobrands/northamptonshire/assets.js

```diff
@@ -617,49 +617,21 @@ function ncc_is_emergency_category() {
     return {relevant: relevant, currently_shown: currently_shown, body: body};
 }
```

Comment:
Presumably this should be removed in this commit and not the next.

Reply:
Can't easily tell what this is referring to sorry; if it's ncc_is_emergency_category, that is still called somewhere, e.g. asset_found, until the last commit of this PR (when it's removed)?

Reply:
Yes, so it is. I missed that.